deploy/helm/scf/values.yaml

```diff
@@ -1,6 +1,14 @@
 system_domain: ~
 deployment_name: scf
 
+# Set or override job properties. The first level of the map is the instance group name. The second
+# level of the map is the job name. E.g.:
+#   properties:
+#     adapter:
+#       adapter:
+#         scalablesyslog.adapter.logs.addr: scf-log-api:8082
```

Comment:
Can we make this a deep object instead of using the `.` notation?

Reply:
I'll see if it's feasible using the Sprig library that Helm exposes to us.